x-pack/plugins/beats/server/rest_api/tokens/create.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Joi from 'joi';
+import { get } from 'lodash';
+import { CMServerLibs } from '../../lib/lib';
+import { wrapEsError } from '../../utils/error_wrappers';
+
```

Comment:
You might want to carry over the TODOs from the old file about the license check pre-hook and the Kibana audit log, just so you don't forget to implement those for this route.

Reply:
I just checked and the same applies to many of the other routes too.

Reply:
fixed